actionview/lib/action_view/cache_expiry.rb

```diff
@@ -51,6 +52,11 @@ def rebuild_watcher
           end
         end
 
+        def rebuild_watcher
```

Comment:
The method name is no longer reflecting what it does. Given it's private API, might as well update it.

Reply:
To be honest I'm not sure what would be a better name. Technically it does rebuild a watcher because you can only rebuild something it it exists and current implementation returns if `@watched` doesn't exist.

I think it kind of makes sense in a callback/hook context - https://github.com/rails/rails/blob/83a0132af812ecda0c1b17286bc6240b8d37e3fd/actionview/lib/action_view/cache_expiry.rb#L15

We could get rid of it and just move the check directly to the hook which will call `build_watcher` if that's a better option

Reply:
`rebuild` suggest to me that we're throwing away the existing one if it exist.

Maybe `init`? It's nitpick territory, not a deal breaker to me.

Reply:
>rebuild suggest to me that we're throwing away the existing one if it exist.

We actually do. It's not clear but existing `@watcher` will be substituted inside `build_watcher`:
https://github.com/rails/rails/blob/99acf3e78470346c9546c1641b846000fcdda27b/actionview/lib/action_view/cache_expiry.rb#L46

We could set `@watcher = nil` right in the `rebuild_watcher` method but it will be unnecessary due to `build_watcher` doing it anyway